cmd/utils/flags.go

```diff
@@ -1968,24 +1967,13 @@ func MakeChain(ctx *cli.Context, stack *node.Node) (chain *core.BlockChain, chai
 	if err != nil {
 		Fatalf("%v", err)
 	}
+
 	var engine consensus.Engine
-	if config.Clique != nil {
-		engine = clique.New(config.Clique, chainDb)
-	} else {
-		engine = ethash.NewFaker()
-		if !ctx.GlobalBool(FakePoWFlag.Name) {
-			engine = ethash.New(ethash.Config{
-				CacheDir:         stack.ResolvePath(ethconfig.Defaults.Ethash.CacheDir),
```

Comment:
The cacheDir and datasetDir are previously resolved under the root directory, if we use the default config directly, then the path is changed, right?

Reply:
Ah, so the dirs are all initialized with default paths here https://github.com/ethereum/go-ethereum/blob/master/eth/ethconfig/config.go#L106

So I think I have nothing to against